source/support/mapclickarty.sqf

```diff
@@ -1,60 +1,81 @@
-// _null = [salvos, radius, interval between salvos(in seconds), shots per salvo, support type, CP cost, ammotype] execVM "support\arty.sqf"
-//  AMMO TYPES: R_60mm_HE / R_80mm_HE /Bo_Air_LGB(no sounds)/ Grenade / Bo_Mk82
-//  SUPPORT TYPE: 1 = arty, 2 = mortar, 3 = jdam
+/*
+    File: mapclickarty.sqf
 
-_salvos = _this select 0;
-_radius = _this select 1;
-_interval = _this select 2;
-_rps = _this select 3;
-_supportype = _this select 4; // 1 = arty, 2 = mortar, 3 = jdam
-_cost = _this select 5; 
-_ammotype = _this select 6;
+    Author: Kibot
 
+    Description: 
+        Guards and determines the position of a strike, based on location provided by the player, for arty.sqf
 
+    Parameter(s):
+        _this select 0 - NUMBER - Number of ordinance salvos
+        _this select 1 - NUMBER - Radius of 'Strike Area'
+        _this select 2 - NUMBER - Interval between salvos (in seconds)
+        _this select 3 - NUMBER - Number of ordinance rounds per salvo
+        _this select 4 - NUMBER - Support type enumerator (1 - Artillery, 2 - Mortar Shell, 3 - JDAM)
+        _this select 5 - NUMBER - Cost of Support type
+        _this select 6 - STRING - Ammo type used for salvos
 
+    Returns: 
+        - Nil -
+*/
+
+params["_salvos", "_radius", "_interval", "_rps", "_supportype", "_cost", "_ammotype"];
 _timer = 60;
+_pos = [];
 clicked = false;
```

Comment:
Does this really have to be a global variable? It has a very generic name. Chances are high that somebody else also uses that name. That's why you should use OFPEC tags.